### ISSUE TITLE
Multiple Active Slide Links

### ISSUE THREAD

**Issue by teksuprc** [ekallevig/jShowOff#53]
I'm using Ajax to create slides. I have four buttons that make four separate Ajax calls and update slide content. After each web call (one at a time) I use jquery to empty the container (parent element) and feature (jshowoff element) then recreated the slides and recreate the jshowoff element.

The initial load works fine, but any other load after keeps the active slide links rotating from the previous load. I have crazy active slide link rotations. I'm looking for either a work around or a fix.

// html

<div id="wrap">
    <div id="container">
       <div id="feature"></div>
    </div>
</div>

// js
function create()
{
    $('<div id="container">').appendTo('#wrap');
    $('<div id="feature">').appendTo('#container');

```
// create the slides and append to the passed object
createSlidesAndAppend('#feature');

$('feature').jshowoff({    
  speed: 1200,
  effect: 'fade',
  links: true,
  controls: false
});
```

}

To sum up: Initial page loading active slide links rotate fine. After making an Ajax call to change the slides and emptying the "container" and "feature" divs, recreating the "container", "feature" and slides using jquery I get multiple "ACTIVE" slide links rotating.
